Code/Engine/RendererCore/Pipeline/Passes/SourcePass.h

```diff
@@ -2,6 +2,28 @@
 
 #include <RendererCore/Pipeline/RenderPipelinePass.h>
 
+struct ezSourceFormat
+{
+  using StorageType = uint8_t;
+
+  enum Enum
+  {
+    Color4Channel8BitNormalized_sRGB,
+    Color4Channel8BitNormalized,
+    Color4Channel16BitFloat,
+    Color4Channel32BitFloat,
+    Color3Channel11_11_10BitFloat,
+    Color3Channel32BitFloat,
```

Comment:
I think we can remove Color3Channel32BitFloat, most Vulkan devices don't support it so it's probably emulated in DX11 by a 4-coponent format anyways. See https://vulkan.gpuinfo.org/listoptimaltilingformats.php and search for r32